termenv_unix.go

```diff
@@ -30,12 +30,20 @@ func colorProfile() Profile {
 		return ANSI256
 	}
 
+	switch term {
+	case "xterm-kitty":
+		return TrueColor
+	}
+
 	if strings.Contains(term, "256color") {
 		return ANSI256
 	}
 	if strings.Contains(term, "color") {
 		return ANSI
 	}
+	if strings.Contains(term, "ansi") {
+		return ANSI
+	}
```

Comment:
I think this could be in the switch case above... as the string seems to be exactly `ansi`

Reply:
Yeah, had the same thought at first, but moved it down eventually: if it's `foobar-ansi`, it's probably still correct to assume it's supporting ANSI colors.